code/__HELPERS/priority_announce.dm

```diff
@@ -170,7 +81,7 @@
 		title = "Classified [command_name()] Update"
 
 	if(announce)
-		priority_announce("A report has been downloaded and printed out at all communications consoles.", "Incoming Classified Message",  "commandreport", has_important_message = TRUE)
+		priority_announce("A report has been downloaded and printed out at all communications consoles.", "Incoming Classified Message", "commandreport", SSstation.announcer.get_rand_report_sound(), has_important_message = TRUE)
```

Comment:
These args are malformed; command report sounds are muted as a result ("commandreport" is no longer an announcer event sound).